server/routes/bookedtoday/choosePrisonerController.test.ts

```diff
@@ -206,7 +207,7 @@ describe('GET /confirm-arrival/choose-prisoner/:id', () => {
           pncNumber: '01/123456',
           fromLocationType: LocationType.COURT,
           isCurrentPrisoner: true,
-          potentialMatches: [],
+          matchType: MatchType.SINGLE_MATCH,
```

Comment:
these tests now all rely on logic derived from the match types rather than the number of potential matches and prisonNumber and Pnc